### ISSUE TITLE
Petros deploys to 0,0,0

### ISSUE THREAD

**Issue by Toakan** [official-antistasi-community/A3-Antistasi#1763]
**Describe the bug**
It appears to be possible for Petros to still deploy himself down in 0,0,0. This occurred on the Test server Wednesday night and  all assets disappeared.

This required Admins to reset Petros.

**To Reproduce**
Cannot determine what happened.

**Expected behaviour**
Petros should deploy to marked position.

**Environment (please complete the following information):**
Version: 2.4
Map (specify if Altis Blufor): Sahari

* [ ] LAN Hosted
* [ ] Private dedicated server
* [ ] Private dedicated server with Headless client(s)
* [ ] Community Server 1
* [ ] Community Server 2
* [X] Community Test Server
* [ ] Community Test Server 2

Modset:
    
Time the bug occurred (Please leave in GMT+0):
19:45 GMT

**Additional context**

RPT File
[arma3server_x64_2021-03-03_11-35-56.txt](https://github.com/official-antistasi-community/A3-Antistasi/files/6090963/arma3server_x64_2021-03-03_11-35-56.txt)

**Comment by MeltedPixel** [comment]
I don't believe this happens anymore. If it does, we can re-open with a new issue.
